src/app/public/testing/dropdown/dropdown-fixture-dropdown.ts

```diff
@@ -0,0 +1,13 @@
+import {
+  SkyDropdownHorizontalAlignment
+} from '@skyux/popovers/modules/dropdown/types/dropdown-horizontal-alignment';
```

Comment:
This should just be `@skyux/popovers` yes?

Reply:
Unfortunately, no... we didn't export this type in our public api. I added the export to this PR, so we'll have to circle back around and change this line after we release.

Reply:
That deep import shouldn't even work to begin with...? We might need to release the horizontal alignment type first, before your PR is merged.